SConstruct

```diff
@@ -5,7 +5,7 @@ DEBUG = ARGUMENTS.get("debug", 0)
 
 CUDA_PATH = os.environ['CUDA_PATH']
 CUDA_INCLUDE_PATH = CUDA_PATH + "/include"
-OPTIX_PATH = "C:/ProgramData/NVIDIA Corporation/OptiX SDK 7.6.0"
+OPTIX_PATH = "/home/baoxiaohe/NVIDIA-OptiX-SDK-8.0.0-linux64-x86_64"
```

Comment:
This looks like a local path on your machine right? Is there a default install directory on linux for OptiX? If not, maybe we should make this an argument for scons?

Reply:
I have added some instructions on README.md and don't change the SConstruct file.